src/views/tickets/rsvp.php

```diff
@@ -143,4 +152,8 @@
 $content = ob_get_clean();
 if ( $is_there_any_product ) {
 	echo $content;
+}else{
```

Comment:
@borkweb I dont mind putting it down, so if its better for you i'll take the lines down today evening :)